sse2neon.h

```diff
@@ -126,13 +126,22 @@
 #error \
     "You must enable NEON instructions (e.g. -mfpu=neon-fp-armv8) to use SSE2NEON."
 #endif
+#if !defined(__clang__)
 #pragma GCC push_options
+#endif
 #else
 #error "Unsupported target. Must be either ARMv7-A+NEON or ARMv8-A."
 #endif
 #endif
 
 #include <arm_neon.h>
+#if !defined(__aarch64__) && (__ARM_ARCH == 8)
+#if defined __has_include
```

Comment:
Rewrite as the following for consistency.
```c
#if defined(__has_include) && __has_include(<arm_acle.h>)
#include <arm_acle.h>
#endif
```

Reply:
I did it that way because https://gcc.gnu.org/onlinedocs/cpp/_005f_005fhas_005finclude.html claimed combining the tests into a single expression wouldn't work in environments where __has_include wasn't available.  But GCC also says && in the preprocessor follows the same short-circuit rules as C, so I don't know why they'd claim it wouldn't work.  I'll change it.